test/selenium_tests/test_jupyter.py

```diff
@@ -1,77 +0,0 @@
-from base.populators import flakey
```

Comment:
well that isn't right... are these just failing locally (that is kind of expected)

Reply:
I don't know, I keep getting a varying number of failing tests since the last 2-3 commits for the last few days and now I am trying to disable all failing tests first to get an overview.